topology/platform/common/ssp.m4

```diff
@@ -43,4 +43,8 @@ define(`SSP_CONFIG_DATA',
 `}'
 )
 
+dnl SSP dai have loopback mode so set the SSP_BESPOKEN to 1
+
+define(`SSP_BESPOKEN_CONFIG', 1)
```

Comment:
This should be SSP_BESPOKE_CONFIG and should define vendor tuples and vendor data e.g

define(`SSP_BESPOKE_CONFIG',
`SectionVendorTuples."'N_DAI_OUT($2)`_tuples_w_comp" {'
`	tokens "sof_comp_tokens"'
`	tuples."word" {'
`		SOF_TKN_COMP_PERIOD_SINK_COUNT'		STR($5)
`		SOF_TKN_COMP_PERIOD_SOURCE_COUNT'	STR($6)
`		SOF_TKN_COMP_PRELOAD_COUNT'		STR($7)
`	}'
`}'
`SectionData."'N_DAI_OUT($2)`_data_w_comp" {'
`	tuples "'N_DAI_OUT($2)`_tuples_w_comp"'
`}'

Reply:
Yes, the idea is good. The SectionData and SectionVendorTuples is easy to create.
If we need to add SectionData into the SectionWidget, it will need more ifelse to check if we have such SectionData. 
Is it OK to have more ifelse?